### PR TITLE
[hub] Replace TSLint with ESlint in Hub

### PR DESCRIPTION
**Pull request by Alonski**
### Description

This PR replaces TSLint with ESlint in Hub.
TSLint is deprecated.

The rules can be changed.
What does everyone think?

### CODE REVIEW

**Comment by kerzhner** [comment]
The removal of `packages/channel-provider/tslint.json` probably doesn't belong in this PR.

File: packages/hub/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+/coverage/
+!.*
+/.vscode/
+/typings
+/migrations/
+**/*.json
+/website/
+/.etherlime-store/
+/lib/
+/dist/
```
Comment (kerzhner) [inline packages/hub/.eslintignore]: Nit, should this contain a newline at the end of the file?

Comment (Alonski) [reply packages/hub/.eslintignore]: I am on Windows 😬
Should we enforce line endings?

Comment (kerzhner) [reply packages/hub/.eslintignore]: Ah gotcha. It might be nice to enforce a newline at the end of files.

**Pull request merged**